### ISSUE TITLE
FUN COMMAND: Make stroke translate command

### ISSUE THREAD

**Issue by notsniped** [PyBotDevs/isobot-lazer#61]
# The stroke translate command
`/stroke-translate` command will let users have some fun as you can translate "gibberish" to a string full of random words! The meaning of the sentence or the word doesn't have to make sense, but it might as well.  

## Proposed usage
Command:
```py
await stroke_translate(ctx:SlashContext, word:str)
```
Output:
```py
return f"{translated} " # For every single letter in input
```

## Pending tasks
- [ ] Make a dictionary with each letter having a word assigned to it (there can be multiple)
- [ ] Make the command in Program file
- [ ] Test and debug command

**Comment by notsniped** [comment]
The letter-to-word dictionary should be something like this maybe?
```json
{
    "i": [
        "isobot",
        "isobot-lazer"
    ],
    "g": [
        "github"
    ]
}
```

**Comment by archisha69** [comment]
dood just pull it from iso6.9 if u want and just add the words

**Comment by notsniped** [comment]
~~time to copy code~~

**Comment by notsniped** [comment]
Merged #70
